utils.ts

```diff
@@ -293,8 +293,20 @@ export async function bisectVite(
 	good: string,
 	runSuite: () => Promise<Error | void>
 ) {
+	const revertChanges = async ()=>{
+		// sometimes vite build modifies files in git, eg. LICENSE.md
+		// this would stop bisect, so to reset those changes
+		// stash them and drop the stash
+		try {
+			await $`git stash`
+			await $`git stash drop`
```

Comment:
I think you could possibly do this with just one command instead of two
```suggestion
			await $`git reset --hard HEAD`
```